vllm/config.py

```diff
@@ -886,12 +899,19 @@ def __init__(
             draft_parallel_config: ParallelConfig for the draft model.
             num_speculative_tokens: The number of tokens to sample from the
                 draft model before scoring with the target model.
+            speculative_disaable_by_enqueue_requests: Disable speculative
+                decoding for new incoming requests when the number of
+                enqueue requests is larger than this value.
```

Comment:
The code in this PR disables spec decode when the _batch size_ is larger than this value, not enqueued request count.

Is this intentional?

Reply:
This is what we did in the internal fork but I'm not sure which one is better (enqueued requests vs. batch size). But I guess I'll just change it to `batch-size` for now to be clear.